conjure-java-core/src/main/java/com/palantir/conjure/java/util/Javadoc.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.util;
+
+import com.palantir.conjure.spec.Documentation;
+import org.apache.commons.lang3.StringUtils;
+import org.commonmark.node.Paragraph;
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.Renderer;
+import org.commonmark.renderer.html.CoreHtmlNodeRenderer;
+import org.commonmark.renderer.html.HtmlRenderer;
+
+/** Provides utility functionality for rendering javadoc. */
+public final class Javadoc {
+
+    private static final Parser parser = Parser.builder().build();
+    private static final Renderer renderer = HtmlRenderer.builder()
+            // Escaping existing HTML may break docs that rendered fine previously.
+            .escapeHtml(false)
+            // Disable paragraph tags, prefer raw text instead. Otherwise
+            // all javadoc will be wrapped in paragraphs.
```

Comment:
Does javadoc respect line breaks? If not, think you have to have the paragraph tags (and would ideally skip the very first set).

Reply:
The rendered javadoc/html does not respect line breaks, however applying paragraph tags would be a significant change to existing documentation in ways that may or may not make it more readable, for example if it already uses html tags specifically for javadoc.
My goal for this change was to provide support for the format we suggest in the specification without significant changes to existing generated code, for now.

Reply:
Added support for line breaks.